backend/src/openarchiefbeheer/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -1423,7 +1423,7 @@ msgstr[1] ""
 #: openarchiefbeheer/logging/templates/logging/destruction_list_co_reviewers_added.txt:9
 #, python-format
 msgid "They also removed these co-reviewers: %(removed_co_reviewers)s."
-msgstr ""
```

Comment:
Check dit ff met @SilviaAmAm, volgens mij wordt hier niet meervoud bedoeld.

Reply:
Dit is inderdaad de ["singular they"](https://en.wikipedia.org/wiki/Singular_they)